cluster/gce/gci/configure-helper.sh

```diff
@@ -1298,6 +1298,9 @@ function prepare-kube-proxy-manifest-variables {
   if [[ -n "${FEATURE_GATES:-}" ]]; then
     params+=" --feature-gates=${FEATURE_GATES}"
   fi
+  if [[ "${KUBE_PROXY_MODE:-}" == "ipvs" ]];then
+    params+=" --proxy-mode=ipvs --feature-gates=SupportIPVSProxyMode=true"
+  fi
```

Comment:
I think can load kernel modules here.

Reply:
Do we want to load kernel modules on demand when this flag is set or does it make more sense to make sure that they are already loaded on the OS being used? How long does it take to load the modules? How will it impact node startup latency?

Reply:
I will defer to @m1093782566 regarding the latency of loading the modules, although it does seem very fast on the surface. 

Regarding the option of making sure they are already loaded on the OS: the user could theoretically choose a node image which doesn't already have the modules loaded, right? Wouldn't we then have to make sure every possible image the user can choose has the modules loaded? If loading is fast, I'm not sure its worth it.

Reply:
> How long does it take to load the modules? How will it impact node startup latency?

In my experience, loading ip_vs_* kernel modules is very fast and users would not feel that.

Reply:
Loading IPVS modules should be OK wrt to latency.